.github/workflows/analyze.yml

```diff
@@ -43,7 +55,7 @@ jobs:
     - name: Remove files in results bucket
       run: aws s3 rm  s3://cellpack-results --recursive
```

Comment:
We should get rid of this step if we're taking care of this in cleanup

Reply:
yea good call